assignment-2/7s.md

```diff
@@ -1,2 +1,238 @@
+The Spotify technology career framework, consist on letting the employee transfer itself on as many roles and domain expertise as possible in order to gain experience and later evaluate their impact into the organization.
+*On recognition and promotions* Spotify observes constantly the impact someone has at a wider scope, and they have a formal ways to promote their employees
+*On setting the step for a new employee* Spotify gives 6 months to all new employees to get involved with their role and the organization.
+*On compensation* A formal promotion should lead to higher pay in a straightforward and transparent way.
+
+##### Skills
+
+> Skills are the special, distinctive competencies of the organization, both in the people and the
+> organization itself. What is the organization good at, and what special skills and competencies
+> are found its people?
+
+What is the organization good at?
+Spotify seems to be good at keeping their organization dynamic, focusing on their community rather to an hierarchical structure.
+
+##### Style
+
+> Style deals mainly with the leadership approach of the top managers in the organization. How do
+> the leaders interface with subordinates and others in the organization, and how do members
+> interact with each other? Style also incorporates the climate or culture of the organization, which
+> can be expressed in terms of job titles, space allocation, workplace architecture, artifacts, and
+> policies.
+
+How do the leaders interact with subordinates and others in the organization?
+According with Spotify [12]:
+- They have an Agile engineering culture, which means they like servent leaders rather than process masters.
+- Autonomy that every member team has over their work have evolve to a high rate percentage of team members satisfaction and they keep investing on it. 
+- Autonomy seems also to keep high motivation levels [12 and 13], teams are fast, and their minimize de dependency from and to, other teams.
+ 
+##### Shared Values
+
+> Shared values are the core beliefs in the organization about what is important and why the
+> organization exists. They are incorporated in formal statements, such as a vision statement, and
+> in informal expressions and practices around the organization. Shared values are the underlying
+> principles or ideas that provide meaning and purpose in an organization.
+
+The vision of Spotify can best be decribed in the following manner: give people access to all the music they want all the time, in a completely legal manner. It is their mission to ensure a fair digital deal for artists.
+
+Adding more functional and visionary features to their software product.
+Keeping the organization as a very nice place to work in.
+
+
+## 2.2 Non-Software industry ##
+
+### KLM
```

Comment:
I commented on this yesterday, but seems that that comment is disappeared. To repeat:

I think it is not a good choice to keep KLM in. What we have now is very limited. Without further references it is not possible to analyze it. Even though it is late, it is better to change now and still write something limited, but useful.

I proposed Heineken yesterday. For example as starting paper: http://www.emeraldinsight.com/doi/pdfplus/10.1108/09534810210429336

I have to look into it more to see whether it is feasible to analyze it. I'll try to discuss this with you somewhere this evening. If you have better suggestions, please let me know ASAP.

The information of Spotify (and KLM) is now too limited to reflect on HealthCare.gov. So we have to progress more om these topics first.